.azure-pipelines/testbed_health_check.py

```diff
@@ -303,22 +310,40 @@ def check_bgp_session_state(self, state="established"):
         logger.info("======================= check_bgp_session_state starts =======================")
 
         for sonichost in self.sonichosts:
+            if (self.is_multi_asic and
+                    self.duts_basic_facts[sonichost.hostname]["ansible_facts"]["dut_basic_facts"]["is_supervisor"]):
```

Comment:
Do we have multi-asic T1 in our lab? If we have, ["is_supervisor"] may report error

Reply:
Yes, good catch! We should also consider the multi-asic T1 scenario.

Directly using ["is_supervisor"] will actually **not** report any error because we are giving a default value of `False` in `dut_basic_facts.py` [here](https://github.com/sonic-net/sonic-mgmt/blob/master/ansible/library/dut_basic_facts.py#L46). However, we should still separate T2 chassis and multi-asic T1 because, for example, multi-asic T1 does not have the `multi_asic.get_asic_presence_list()` method as it does not have the "logical" concept like T2 chassis. Besides, there might be more features adding to T2 chassis and multi-asic T1 in the future, which might make them more different between each other.

Therefore, I added a `is_chassis` property in `dut_basic_facts.py` so we can treat them separately whenever needed. I have verified this change on single-asic Testbed, mutli-asic T1 Testbed & T2 chassis, and I can confirm it's working on all of them. (After this PR, health checks on multi-asic Testbeds will still be disabled/skipped)